client/src/extension.ts

```diff
@@ -89,8 +90,26 @@ export function activate(context: ExtensionContext) {
         ExecYseopCliCommand(yseopCliPath, 'package');
     });
 
-    const infoCmd = commands.registerCommand(`${yseopmlSectionName}.info`, () => {
-        ExecYseopCliCommand(yseopCliPath, 'info');
+    const infoCmd = commands.registerCommand(`${yseopmlSectionName}.config`, () => {
+        ExecYseopCliCommand(yseopCliPath, 'config');
+    });
+
+    const libsInstallCmd = commands.registerCommand(`${yseopmlSectionName}.libs`, () => {
+        ExecYseopCliCommand(yseopCliPath, 'libs', 'install', '-R');
+    });
+
+    const deployCmd = commands.registerCommand(`${yseopmlSectionName}.managerDeploy`, (ymaUri: Uri) => {
+        window
+            .showInputBox({ prompt: 'Leave this box empty to use the default env.', placeHolder: 'DEMO' })
+            .then((env) => {
+                if (!!env && env.length > 0) {
+                    window.showInformationMessage(`Deploying YMA ${ymaUri.fsPath} on env “${env}”.`);
+                    ExecYseopCliCommand(yseopCliPath, 'manager', 'deploy', '-e', `${env}`, ymaUri.fsPath);
```

Comment:
`-e ''` is accepted as an equivalent of “no -e at all”, because I do “if is not blank” and stuff. So your `if` seems a bit overkill.

Reply:
> So your if seems a bit overkill.

I prefer to have a case where we have a nice message saying that we're using the default environment.
I'll let the `if` statement as it is.

Reply:
For the message, sure, but then why duplicate the command? You could do

```
if …
  message
else
  other message

command
```

or use a ternary operator or whatever. But having two `ExecYseopCliCommand` here seems useless. That was my point.

Reply:
Alright!
Done.